### PR TITLE
added custom actions url

### PR DESCRIPTION
**Pull request by desmarchris**
**Proposed changes**:
- I had forgotten to uncomment the URL in `endpoints.yml` when building out my demo, so I basically moved part of the 1.x docs to 2.x: https://legacy-docs-v1.rasa.com/core/actions/#custom-actions. Is there a reason this was left out? Also, I'm open to this being in the sdk docs, but I felt like it was better here to make sure people don't miss it.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/custom-actions.mdx [hunk 1]
```diff
@@ -9,6 +9,15 @@ For details on how to implement a custom action, see the [SDK documentation](htt
 Any custom action that you want to use in your stories should be added into the
 actions section of your [domain](./domain.mdx).
 
+Rasa will call an endpoint you specify when a custom action is predicted.
```
Comment (b-quachtran) [inline docs/docs/custom-actions.mdx:12]:
```suggestion
Rasa will send a request to the actions server when a custom action is predicted. 
```

File: docs/docs/custom-actions.mdx [hunk 2]
```diff
@@ -9,6 +9,15 @@ For details on how to implement a custom action, see the [SDK documentation](htt
 Any custom action that you want to use in your stories should be added into the
 actions section of your [domain](./domain.mdx).
 
+Rasa will call an endpoint you specify when a custom action is predicted. 
+This endpoint should be a webserver that reacts to this call, runs the code, and
```
Comment (b-quachtran) [inline docs/docs/custom-actions.mdx:13]:
```suggestion
The actions server is a web server that responds to this request by running custom code and
```

File: docs/docs/custom-actions.mdx [hunk 3]
```diff
@@ -9,6 +9,15 @@ For details on how to implement a custom action, see the [SDK documentation](htt
 Any custom action that you want to use in your stories should be added into the
 actions section of your [domain](./domain.mdx).
 
+Rasa will call an endpoint you specify when a custom action is predicted. 
+This endpoint should be a webserver that reacts to this call, runs the code, and 
+optionally returns information to modify the dialogue state.
```
Comment (b-quachtran) [inline docs/docs/custom-actions.mdx:14]:
```suggestion
optionally returning information to update the dialogue state.
```

File: docs/docs/custom-actions.mdx [hunk 4]
```diff
@@ -9,6 +9,15 @@ For details on how to implement a custom action, see the [SDK documentation](htt
...
+
+Specify your action server in `endpoints.yml`:
```
Comment (b-quachtran) [inline docs/docs/custom-actions.mdx:16]:
```suggestion
You can specify your action server's endpoint in `endpoints.yml`:
```

**Review by b-quachtran — CHANGES_REQUESTED** [review]
Thanks for adding this! Just a few changes but looks good otherwise

**Comment by desmarchris** [comment]
@b-quachtran thanks for the review! I accepted your changes, lmk if I need to do anything else :)

**Review by b-quachtran — APPROVED** [review]

**Pull request merged**